core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/view/PoolableElementView.java

```diff
@@ -534,6 +536,14 @@ public void setSample(Sample sample) {
     this.sample = sample;
   }
 
+  public Boolean getLibraryQcPassed() {
+    return libraryQcPassed;
+  }
+
+  public void setLibraryQcPassed(Boolean libraryQcPassed) {
+    this.libraryQcPassed = libraryQcPassed;
+  }
+
   @Override
   public int hashCode() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 39 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)